ISISComputingGroup/EPICS-DeviceEmulator
# Fix tests

Fixes issues introduced by Python 3 conversion.

# To test
* Run the maps and merlin fermi chopper tests and confirm they pass
* un the fzj_chopper tests and confirm they pass

Note:
I saw `fermichopper_merlin.MerlinFermiChopperTests.test_WHEN_electronics_temperature_is_too_high_THEN_switch_drive_off_is_sent` fluctuate on my machine. We should fix it but I would suggest that we merge this to at least be able to see the wood through the trees...
